docs/section19/19-19.md

```diff
@@ -0,0 +1,111 @@
+                    - Calculate the minimum BHP: `minimum_BHP = (0.00062 * supply_cfm_b)`
+
+                - Get the motor efficiency from Table G3.9.1 for expected BHP (should be looked up based on the next motor size greater than the bhp (i.e., expected_BHP)): `motor_eff_ex = data_lookup(G3.9.1,expected_BHP)`  
+                - Get the motor efficiency from Table G3.9.1 for minimum BHP (should be looked up based on the next motor size greater than the bhp (i.e., expected_BHP)): `motor_eff_min = data_lookup(G3.9.1,minimim_BHP)`  
+                - Calculate the expected Wattage for expected BHP (assuming no other pressure drop adjustments apply except energy recovery and MERV filters): `fan_Wattage_expected = expected_BHP * 746 *(1/motor_eff_ex)`  
+                - Calculate the minimim Wattage for minimim BHP: `fan_Wattage_minimim = minimim_BHP * 746 *(1/motor_eff_min)`  
+
+
+            - **Rule Assertion:** 
+            - Case 1: If fan wattage expected equals total fan power: `if more_than_one_exhaust_fan_and_energy_rec_is_relevant == false and more_than_one_supply_fan == false and total_fan_power == fan_Wattage_expected: outcome = "PASS"`  
+            - Case 2: Elif conservative comparison equals true and fan is less than expected outcome is Pass:`elif more_than_one_supply_fan == false and AHJ_RA_compare == True and total_fan_power < fan_Wattage_expected: PASS and raise_message "The total fan power for <insert hvac.id> is modeled as <insert total_fan_power> kW which is less than the expected including pressure drop adjustments for exhaust air energy recovery and MERV filters as applicable which was calculated as <insert fan_Wattage_minimim> kW. Pass because this is generally considered more conservative."`  `  
+            - Case 3: Elif conservative comparison is false and fan is less than expected outcome is Fail:`elif more_than_one_supply_fan == false and AHJ_RA_compare == False and total_fan_power < fan_Wattage_expected: "FAIL" and raise_message "The total fan power for <insert hvac.id> is modeled as <insert total_fan_power> kW which is less than the expected including pressure drop adjustments for exhaust air energy recovery and MERV filters as applicable which was calculated as <insert fan_Wattage_minimim> kW ."`         
+            - Case 4: Elif conservative comparison equals true and fan wattage is less than the minimum possible then Pass: `elif more_than_one_supply_fan == false and AHJ_RA_compare == True and total_fan_power < fan_Wattage_minimum: outcome = "Pass" and raise_message "The total fan power for <insert hvac.id> is modeled as <insert total_fan_power> kW which is less than the minimum possible (i.e., less than the expected fan power assuming no pressure drop adjustments) which was calculated as <insert fan_Wattage_minimim> kW. Pass because this is generally considered more conservative."`              
```

Comment:
Wouldn't that be covered be Case 2?

Reply:
@mkarpman Yes but I was not sure why it was suggested that this be calculated (the minimum expected Wattage) during the last round of comments.  Looks like I missed something. Can you please clarify?

Reply:
I suggested adding the minimum fan power so that we can produce FAIL if conservative outcome is not allowed and modeled fan power is less than minimum. So I don't think we need case 4 because it's redundant to Case 2.
So I think the logic should be as follows:
1. If no more than 1 supply fan AND no more than one exhaust fan if energy recovery is relevant AND fan power as expected then PASS
2. If no more than 1 supply fan AND conservative comparison allowed AND fan power less than expected then PASS
3. If no more than 1 supply fan AND conservative comparison not allowed AND fan power less than expected then FAIL
4. If no more than 1 supply fan AND conservative comparison allowed AND fan power less than expected then UNDERTERMINED
5. If no more than 1 supply fan AND conservative comparison allowed AND fan power less than minimum possible then PASS.
6. If no more than 1 supply fan AND conservative comparison not allowed AND fan power less than minimum possible then FAIL.
7. If more than 1 supply fan OR more than one exhaust fan if energy recovery is relevant then UNDERTERMINED

Reply:
Use this as primary reference for making changes.

Reply:
@mkarpman I am not understanding #4. Its the same as 2 but with a different outcome. Can you please explain?  

Also, similar to what you say directly above the list aren't 2 and 5 redundant? If it passes 2 it will pass 5.